reference/testing/README.md

```diff
@@ -4,7 +4,7 @@ FLAT provides a built-in test framework. It is especially useful to unit-test [t
 
 ## Syntax
 
-FLAT tests are defined as [flow](/reference/flow.md) files with the root node `flat-test`. There are a couple of specialized [test actions](/reference/actions/README.md#testing) that accompany the test flow. The invocation of at least one [`assert` action](/reference/actions/assert.md) is mandatory.
+FLAT tests are defined as [flow](/reference/flow.md) files with the root node `flat-test`. There are a couple of specialized [test actions](/reference/actions/README.md#test-actions) that accompany the test flow. The invocation of at least one [`assert` action](/reference/actions/assert.md) is mandatory.
```

Comment:
In anderer MD-Doku haben wir bisher explizite Sprungziele erzeugt, auf die referenziert wird. `#test-actions` scheint mir ein automatisch aus der Überschrift erzeugtes Sprungziel zu sein.

Reply:
github erzeugt die anchors  lowercase und mit `-` statt allen non-url chars. dafür würde ich keine eigenen anker anlegen, zumal gitbook nicht gut mit inline html umgehen kann.